developer-kit/Hiber.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef HIBER_H_
+#define HIBER_H_
```

Comment:
No more pragma once?

Reply:
This is more of a style choice as pragma once is non-standard but widely supported.